js/index.js

```diff
@@ -1,6 +1,117 @@
+
+//word counter
+let count = 1
+for(let i = 0; i<paragraphs.length; i++){
+    if(paragraphs[i]=== " "){
+        count +=1;
+     
+    }
+  
+}
+console.log(count)   
+
+
+// word "et" counter
+let etCounter = 0;
+for(let i = 0; i< paragraphs.length; i++){
+    if(paragraphs[i] ===" " && paragraphs[i+1] === "e" && paragraphs[i+2]=== "t"){
+        etCounter += 1;
+    }
+}
+
+console.log(etCounter)
+
+//BONUS 2
+let phraseToCheck="roma amor";
+let result2;
+
+for (let i = 0; i <= phraseToCheck.length/2; i++){
+    for (let j = phraseToCheck.length; j >= phraseToCheck.length/2; j--){ 
+        if(phraseToCheck[i]=== " " || phraseToCheck[j] === " "){
+            continue;    
+        }else if (phraseToCheck[i] === phraseToCheck[j]){
+            result2 = true;
+        }else{
+            result2 = false;
+        };
+    };
+ };
+
+
+ if(result2 == true){ 
+    console.log(`${phraseToCheck} is a palindrome!`)
+ } else if(result2 == false){
+    console.log(`${phraseToCheck} is not a palindrome!`)
+ };
```

Comment:
Last iteration is not working properly, try and read the recommended documentation for this iteration :)